base/REPLCompletions.jl

```diff
@@ -152,14 +152,17 @@ end
 # Determines whether method_complete should be tried. It should only be done if
 # the string endswiths ',' or '(' when disregarding whitespace_chars
 function should_method_complete(s::AbstractString)
+    method_complete = false
     for c in reverse(s)
         if c in [',', '(']
-            return true
+            method_complete = true
+            break
         elseif !(c in whitespace_chars)
-            return false
+            method_complete = false
+            break
         end
     end
-    false
+    method_complete
 end
```

Comment:
This is better because the function only has one return point instead of three. Before under normal circumstances the two return statements inside the loop would be used. The false in the end was just a precaution I made if the two return points inside the loop was missed for some reason. To test this precaution to get coverage you would need to hand the function an empty string, which the function would never due to other circumstances. I thought this test did not make sense and I liked the idea of having one return point instead of three. Should I change it back?